e2e/playwright/test-network-and-connection-issues.spec.ts

```diff
@@ -98,8 +100,10 @@ test.describe('Test network related behaviors', () => {
     { tag: '@skipLocalEngine' },
     async ({ page, homePage, toolbar, scene, cmdBar }) => {
       const networkToggle = page.getByTestId('network-toggle')
-      const networkToggleConnectedText = page.getByText('Connected')
-      const networkToggleWeakText = page.getByText('Network health (Weak)')
+      const networkToggleConnectedText = page.getByText(
+        'Network health (Strong)'
+      )
+      const networkToggleWeakText = page.getByText('Ok')
```

Comment:
Should this text match the other test's locator? Do we _really_ need to be modifying the tooltip on slower connections? It seems like a decent source of flakiness in tests is having to check for all possible labels that indicate "connected".

```suggestion
      const networkToggleWeakText = page.getByText('Network health (Ok)')
```

Reply:
We check for both in the tests using the `.or` method selector, so no flakiness will come from this.

We logically don't _really_ need to be showing they are even connected, right. Their local network management programs should tell the user if they are connected to a network or not, and how good their connections are, because it's their responsibility to manage the user's network connections.